packages/react-components/react-nav-preview/src/components/NavDrawerBody/NavDrawerBody.types.ts

```diff
@@ -1,17 +1,14 @@
-import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import { DrawerBodySlots, DrawerBodyState } from '@fluentui/react-drawer';
+import type { ComponentProps, ComponentState } from '@fluentui/react-utilities';
 
-export type NavDrawerBodySlots = {
-  root: Slot<'div'>;
-};
+export type NavDrawerBodySlots = ComponentProps<DrawerBodySlots>;
 
 /**
  * NavDrawerBody Props
  */
-export type NavDrawerBodyProps = ComponentProps<NavDrawerBodySlots> & {};
+export type NavDrawerBodyProps = ComponentProps<DrawerBodySlots>;
 
 /**
  * State used in rendering NavDrawerBody
  */
-export type NavDrawerBodyState = ComponentState<NavDrawerBodySlots>;
-// TODO: Remove semicolon from previous line, uncomment next line, and provide union of props to pick from NavDrawerBodyProps.
-// & Required<Pick<NavDrawerBodyProps, 'propName'>>
+export type NavDrawerBodyState = ComponentState<Partial<DrawerBodySlots>> & DrawerBodyState;
```

Comment:
Same for the state
```suggestion
export type NavDrawerBodyState = DrawerBodyState;
```

if we want to keep the name of the type of slots, then we could so something like this:
 ```suggestion
export type NavDrawerBodyProps = ComponentState<NavDrawerBodySlots>;
```